src/mpi/coll/iallgather/iallgather_tsp_brucks.c

```diff
@@ -84,7 +84,8 @@ MPIR_TSP_Iallgather_sched_intra_brucks(const void *sendbuf, MPI_Aint sendcount,
                                  recvcount, recvtype, sched, 0, NULL);
 
     /* All following sends/recvs and copies depend on this dtcopy */
-    MPIR_TSP_sched_fence(sched);
+    mpi_errno = MPIR_TSP_sched_fence(sched);
+    MPIR_ERR_CHECK(mpi_errno);
```

Comment:
Shouldn't we use the `MPIR_ERR_COLL_CHECKANDCONT` macro as in https://github.com/pmodels/mpich/pull/5491?

Actually, are you sure this commit is not covered in #5491 already?

Reply:
I didn't know we have a new macro for error checking. I updated the commit to switch to it,